content/developer/reference/frontend/services.rst

```diff
@@ -692,6 +694,54 @@ When a rpc fails, then:
   displayed and the server is regularly contacted until it responds. The
   notification is closed as soon as the server responds.
 
+.. _frontend/services/scroller:
+
+Scroller service
+----------------
+
+Overview
+~~~~~~~~
+
+- Technical name: `scroller`
+- Dependencies: ``@web/core/utils/scrolling.js``
+
+Provides a way to handle scrolls in the webclient. 
+This service is used when a click occurs on the webclient and can
+trigger appropriate events depending of the click target.
+
+API
+~~~
+The service adds an event listener to get `click`'s on the document. The event is then
```

Comment:
empty line between title and section

Reply:
this text is more a general description. it can be moved above the API sub section